app/components/weatherHome/weatherHome.tsx

```
@@ -0,0 +1,184 @@
+    }
+    return mapData(weatherData.data);
+  };
+
+  useEffect(() => {
+    const getCitiesData = async () => {
+      const currentCityResult = await getWeatherData();
+      setData(currentCityResult);
+      
+      // Sequential calls to avoid rate limiting
+      const cities = ["sydney", "brisbane", "perth", "adelaide"];
+      const otherCitiesResult: WeatherData[] = [];
```

ESLint

[error] 'WeatherData' is not defined.

(no-undef)